tools/single_use/Normalize_Consent_Data.php

```diff
@@ -99,8 +99,8 @@
         $consentID  = $entry['ID'];
         $candID     = $entry['CandID'];
         // Check if consent status is not_answered
-        if($status === "not_answered") {
-            array_push($errors, "Deprecated consent status 'not_answered' found for " . $consentName . ":
+        if($status === "not_answered" || $status === "") {
+            array_push($errors, "Deprecated consent status 'not_answered' or empty string found for " . $consentName . ":
```

Comment:
This still needs to be removed. 

@intralizee Please don't manually dismiss the reviews we've given on your PRs containing this file. The reviews will automatically be resolved when the file is actually removed.

Reply:
Hi @johnsaigle the problem is me setting the branch incorrectly.

Reply:
@johnsaigle I'm not sure if that's true.. in my experience it's hit or miss whether reviews are autodismissed..

Reply:
@driusan Fair enough re the auto-dismissing but I still prefer to resolve my own reviews :man_dancing: